backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/controller/NormWorkController.java

```diff
@@ -0,0 +1,40 @@
+package de.bund.digitalservice.ris.norms.adapter.input.restapi.controller;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import de.bund.digitalservice.ris.norms.adapter.input.restapi.mapper.NormWorkResponseMapper;
+import de.bund.digitalservice.ris.norms.adapter.input.restapi.schema.NormWorkResponseSchema;
+import de.bund.digitalservice.ris.norms.application.port.input.LoadNormUseCase;
+import de.bund.digitalservice.ris.norms.domain.entity.eli.NormWorkEli;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Controller for getting all norms
```

Comment:
But then you should change this description :) (The "but" is because I wrote the other bigger comment first)